src/pages/importAndSaveManyTraineesFromGoogleSheet/importAndSaveManyTraineesFromGoogleSheet.tsx

```diff
@@ -0,0 +1,296 @@
+      const newFormData = {
+        ...dataRetrieved,
+        ...formData,
+      };
+      dispatch(
+        resendMappedDataIntoDb(
+          flipObjectKeys(newFormData),
+          Id_goolge_sheet_back
+        )
+      );
+    } else {
+      localStorage.setItem("formData", JSON.stringify(formData));
+      localStorage.setItem("Id_goolge_sheet", Id_goolge_sheet);
+      dispatch(
+        resendMappedDataIntoDb(flipObjectKeys(formData), Id_goolge_sheet)
+      );
+    }
+  }
+
+  return (
+    <div className="ml-[1rem] mr-[1rem]  h-full p-[9rem] flex justify-center flex-col items-center bg-[#aaa] pb-[20rem]">
+      {!failedStatusMessage && (
+        <form className="w-[80%] h-1/2 ml-[50rem] border-[#c5c5c5] mr-[50rem] bg-slate-50 p-[2rem]  shadow-2xl shadow-blue-100 hover:shadow-indigo-100/40 ">
+          <div className="mb-6">
+            <label
+              htmlFor="email"
+              className="block mb-2 text-sm font-medium text-gray-900"
+            >
+              Enter Google sheet URL
+            </label>
+            <input
+              value={urlInput}
+              onChange={handleChange}
+              type="text"
+              id="email"
+              className="bg-gray-50 border border-gray-300 text-gray-500 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
+              placeholder="Please Enter google sheet url to load your data into database"
+              required
+            />
+          </div>
+          <Link to="/filter_trainee">
+            <button
+              type="submit"
+              className="text-gray-300 mr-4 bg-[#6c1313]  hover:bg-[#931a1a]  focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm w-full sm:w-auto px-5 py-2.5 text-center"
+            >
+              Back to all trainees
+            </button>
+          </Link>
+
+          <button
+            onClick={handleSubmit}
+            className="text-gray-300 bg-button-color hover:bg-[#255d64] focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm w-full sm:w-auto px-5 py-2.5 text-center dark:bg-blue-60"
+          >
+            Load Data to Database
+          </button>
+        </form>
+      )}
+      {failedStatusMessage === "Wrong cycle name is provided!!!!" && (
+        <div>
+          {/* <label htmlFor="" className="">Please enter the correct cycle name which is in Database already!!!</label>
+        <input type="text" /> */}
+
+          <form onSubmit={handleSubmitToResend}>
+            <div className="mb-6">
+              <label
+                htmlFor="text"
+                className="block mb-2 text-sm font-medium text-gray-900 dark:text-white"
+              >
+                Please enter the correct cycle name which is in Database
```

Comment:
Change the message to something simple as @denislohan suggested.
Ex:  `Please enter correct cycle name!`